pkg/monitor/monitor.go

```diff
@@ -135,6 +137,11 @@ func (m *Monitor) EventIntervals(from, to time.Time) monitorapi.EventIntervals {
 	intervals := filterSamples(samples, from, to)
 	events = filterEvents(events, from, to)
 
+	// create additional intervals from events
+	intervals = append(intervals, intervalcreation.IntervalsFromEvents_OperatorAvailable(events, from, to)...)
+	intervals = append(intervals, intervalcreation.IntervalsFromEvents_OperatorProgressing(events, from, to)...)
```

Comment:
I assume you know these don't belong hardcoded here and need to be passed in to cmd_runsuite via Options

Reply:
> I assume you know these don't belong hardcoded here and need to be passed in to cmd_runsuite via Options

moved to where we specify the monitors, so they are paired.